bedrock/privacy/redirects.py

```diff
@@ -20,4 +20,7 @@
 
     # mozilla/bedrock/#5745 - archive cliqz policy
     redirect(r'^privacy/firefox-cliqz/?$', 'privacy.archive.firefox-cliqz-2018-06'),
+
+    # mozilla/bedrock/##7983
+    redirect(r'^/privacy/products/','/firefox/privacy/products/'),
```

Comment:
```suggestion
    redirect(r'^/privacy/products/', 'firefox.privacy.products'),
```

Please also add a redirect test for this. Thanks